posts/2017-01-05-serverless-v1.5.0.md

```diff
@@ -0,0 +1,97 @@
+---
+title: IoT event support, Cognito user pool authorizer, service installation with a name in Serverless Framework v1.5
+description: IoT event support, Cognito user pool authorizer, service installation with a name in Serverless Framework v1.5
+date: 2017-01-05
+layout: Post
+authors:
+  - PhilippMuns
+---
+
+Today we're happy to announce the release of the Serverless Framework v1.5.0.
+
+Let's take a look at the release highlights!
+
+## Highlights of 1.5.0
+
+**Note:** You can find a complete list of all the updates in the [changelog](https://github.com/serverless/serverless/blob/master/CHANGELOG.md).
+
+### IoT event support
+
+v1.5 introduces the new `iot` event source.
```

Comment:
100% agree in that. I'll write a bash script so that we can extract all the contributors automatically and add them in the release blog posts!